Code/GraphMol/SmilesParse/smatest.cpp

```diff
@@ -102,6 +102,8 @@ void testPass() {
     "[z{1-3}]",
     "[Z{1-3}]",
     "[2H,13C]",  // github #1719
+    "[+{0-3}]",
+    "[-{0-3}]",
```

Comment:
Could we try an or as well?

```
"[-{0-3},+{0-3}]"
```

Reply:
This is just checking that the syntax is ok and that the thing can be serialized/deserialized. We could add a functional test with or; do you have some reason to think that it’s likely to be a problem?

Reply:
Nope, just curious.

Reply:
what's hilarious is that `"[-{0-3},+{0-3}]"` actually *is* an error with the current SMARTS parser.

Reply:
actually, combining any two range queries on a single atom leads to an error. Lovely